### PR TITLE
add docker-push to travis

### PR DESCRIPTION
**Pull request by smacker**
Fixes: #33

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,7 @@ target/
 
 metastore_db/
 scylla
-derby.log
\ No newline at end of file
+derby.log
+
+/Makefile.main
+/.ci
```
Comment (dpordomingo) [inline .gitignore]: Unexpected EOF

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Package configuration
+PROJECT = gemini
+
+# Including ci Makefile
+MAKEFILE = Makefile.main
+CI_REPOSITORY = https://github.com/src-d/ci.git
+CI_FOLDER = .ci
```
Comment (dpordomingo) [inline Makefile]: Why are we using this names and import rule? It seems to be compared from somewhere but [the current recommendations](https://github.com/src-d/ci/blob/master/examples/ci/Makefile)

File: .gitignore (hunk 1 above)
Comment (smacker) [reply .gitignore]: how can I reproduce it? travis works just fine

File: Makefile (hunk 2 above)
Comment (smacker) [reply Makefile]: https://github.com/bblfsh/dashboard/blob/master/Makefile#L7
don't see much difference between it and the example. they do the same thing.

**Pull request merged**